apps/demo-ui/ui/.prettierrc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
how does the styling take effect? do we need to setup the idea to use it or is it all automatic?